client/extensions/woocommerce/lib/order-status/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Lists of statuses in each group, waiting for payment, waiting for
+ * fulfillment, and finished orders.
+ */
+export const statusWaitingPayment = [
```

Comment:
It seems it would make it less likely to be broken down the road if we could get your currentStatus possible values defined by this module as well instead of trusting that the strings will be correctly maintained across the modules that use them